packages/admin/admin/src/dataGrid/CrudMoreActionsMenu.tsx

```diff
@@ -27,7 +27,6 @@ export type CrudMoreActionsMenuClassKey = "root" | "group" | "divider" | "button
 export interface ActionItem extends ComponentProps<typeof MenuItem> {
     label: ReactNode;
     icon?: ReactNode;
-    divider?: boolean;
```

Comment:
Can we remove this prop? Isn't this part of the component's public API?

Reply:
It already exists from the extending `ComponentProps<typeof MenuItem>` so it does not need to be defined here as well.